src/main/java/org/example/service/TableService.java

```diff
@@ -0,0 +1,33 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+public class TableService {
+
+    private static final DeveloperService developers = DeveloperService.getInstance();
+
+    private static TableService INSTANCE;
+
+    public String getHtmlTable() {
```

Comment:
Add some info about content of this table in the method name.

Reply:
fixed